README.md

```diff
@@ -43,151 +43,32 @@ ICICLE is a CUDA implementation of general functions widely used in ZKP. ICICLE
 
 ### Prerequisites
 
-- [NVCC] (version 12.0 or newer)
-- cmake 3.18 and above
-- follow [these instructions](https://github.com/ingonyama-zk/icicle/tree/main/icicle#prerequisites-on-ubuntu)
-- Any Nvidia GPU
+- [CUDA Toolkit](https://developer.nvidia.com/cuda-downloads?target_os=Linux&target_arch=x86_64&Distribution=Ubuntu) version 12.0 or newer.
```

Comment:
I would point to https://developer.nvidia.com/cuda-downloads since its possible to run on Windows and potentially other linux platforms